site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -104,4 +105,18 @@ const getDefaultParameterValues = (
   return paramValues
 }
 
+export const orderedTemplateParameters = (
```

Comment:
Is it used in more than one place? If not, I think we can remove the `export` or move this function to live in the file where it is used.

Reply:
It is used in 2 places as it's also in `WorkspaceBuildParametersPage`, so I will keep it as is, but thanks for raising this!